app.js

```diff
@@ -60,37 +37,72 @@ let fetch = (user, repo) => {
       sort: "updated",
       order: "desc",
       since: ghCommit.commit.author.date
-    });
-  }).then((issues) => {
-    const json = {};
+    })
+  }
+}
+
+const testLabel = (l) => { return regex.test(l.name) }
+
+const fetchCommits = (user, repo) => {
+  return github.repos.getCommits({
+    user,
+    repo,
+    sha: "production"
+  })
+}
+
+const collectIssues = (issues) => {
+  return new Promise((resolve, reject) => {
```

Comment:
This whole function doesn't look like it needs to be return a promise (it looks synchronous to me), however a nicer way to achieve the current semantics would be to do:
```js
const collectIssues = Promise.method (issues) => {
  let json = {}
  ...
  return json
}